src/pluto/pluto.py

```diff
@@ -284,11 +284,20 @@ def create_null(df_null, v1, v2, v3, condo, mapped):
                 & (df_null.mapped == mapped)
                 & (df_null.pair.isin([f"{v1} - {v2}", f"{v2} - {v3}"])),
                 :,
-            ]
-            v1v2 = df.loc[df_null.pair == f"{v1} - {v2}", :].to_dict("records")[0]
-            v2v3 = df.loc[df_null.pair == f"{v2} - {v3}", :].to_dict("records")[0]
-            v1v2_total = v1v2.pop("total")
-            v2v3_total = v2v3.pop("total")
+            ].drop_duplicates()
+
+            v1v2_exist=True
+            v2v3_exist=True
+            try:
```

Comment:
the .to_dict("record")[0] is what causes the error - you could instead reorganize this like
```
v1v2_df = df.loc[df_null.pair == f"{v1} - {v2}", :]
v2v3_df = df.loc[df_null.pair == f"{v2} - {v3}", :]

if v1v2_df.empty() and v2v3_df.empty():
  st.info("Stuff about this report not existing"))
  return
if not v1v2_df.empty():
  v1v2 = v1v2_df.to_dict(blah)
  v1v2_total = 
  fig.add_trace(stuff)
# same for v2v3  
```